lambdas/credentialissuerreturn/src/main/java/uk/gov/di/ipv/core/credentialissuerreturn/CredentialIssuerReturnHandler.java

```diff
@@ -105,9 +105,9 @@ public APIGatewayProxyResponseEvent handleRequest(
                             .getUserId());
 
             auditService.sendAuditEvent(
-                    AuditEventTypes.IPV_CREDENTIAL_RECEIVED_AND_SIGNATURE_CHECKED, null);
+                    AuditEventTypes.IPV_CREDENTIAL_RECEIVED_AND_SIGNATURE_CHECKED);
 
-            auditService.sendAuditEvent(AuditEventTypes.IPV_VC_RECEIVED, null);
+            auditService.sendAuditEvent(AuditEventTypes.IPV_VC_RECEIVED);
```

Comment:
It seems odd we have these two events...

Reply:
Yeah good point. Not sure why we'd need both here they seem like the same thing. Plus I can't find anything about the IPV_CREDENTIAL_RECEIVED_AND_SIGNATURE_CHECKED one here in the list of events - https://docs.google.com/spreadsheets/d/1cLjAdRcpw94uYLnrt7FTeFX5XM_7MLRTDWdf-Zbobj8/edit?pli=1#gid=485429246

Reply:
Might be a bit of a cleanup thing to do after to double check that we don't need the 1st one